.tekton/visual-qontract-master-push.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1
```

Comment:
`main` instead of `master` for consistency

Reply:
done